### ISSUE TITLE
Email+ [crash] Requested value 'Warn' was not found

### ISSUE THREAD

**Issue by adamncsu** [datalust/seq-apps#52]
Email+ crashes when I try to send Warning-level events. I'm using NLog, which labels the Warning event 'Warn' instead of 'Warning', which is why I assume it's crashing.

Curiously, the app does NOT throw an error for 'Info' level events, even though the Serilog version is labeled 'Information'.

As a side note, I have no idea how to restart a crashed App. My instance wouldn't send any more emails after it crashed. I had to remove the instance and create a new one, which seems kinda silly. 

Here's the full stack trace:
```
System.ArgumentException: Requested value 'Warn' was not found.
   at System.Enum.EnumResult.SetFailure(ParseFailureKind failure, String failureMessageID, Object failureMessageFormatArgument)
   at System.Enum.TryParseEnum(Type enumType, String value, Boolean ignoreCase, EnumResult& parseResult)
   at System.Enum.Parse(Type enumType, String value, Boolean ignoreCase)
   at Serilog.Formatting.Compact.Reader.LogEventReader.ReadFromJObject(Int32 lineNumber, JObject jObject)
   at Seq.Apps.GenericHost.AppHost.<SendAsync>d__19.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Seq.Apps.GenericHost.Program.<Main>d__0.MoveNext()
```

**Comment by nblumhardt** [comment]
Thanks for the report, Adam! This looks like a Seq bug, I'll raise and link an issue over on that side 👍

**Comment by nblumhardt** [comment]
This is fixed in Seq 4.2.1113.
